base/applications/regedit/resource.h

```diff
@@ -73,48 +72,19 @@
 
 #define ID_HELP_HELPTOPICS            32794
 #define ID_HELP_ABOUT                 32795
-#define ID_WINDOW_CASCADE             32797
-#define ID_WINDOW_TILE                32798
-#define ID_WINDOW_ARRANGEICONS        32799
-#define ID_OPTIONS_FONT               32800
-#define ID_OPTIONS_AUTOREFRESH        32801
-#define ID_OPTIONS_READONLYMODE       32802
-#define ID_OPTIONS_CONFIRMONDELETE    32803
-#define ID_OPTIONS_SAVESETTINGSONEXIT 32804
-#define ID_SECURITY_PERMISSIONS       32805
-#define ID_VIEW_TREEANDDATA           32806
-#define ID_VIEW_TREEONLY              32807
-#define ID_VIEW_DATAONLY              32808
-#define ID_VIEW_DISPLAYBINARYDATA     32810
-#define ID_VIEW_REFRESHALL            32811
-#define ID_VIEW_REFRESHACTIVE         32812
-#define ID_VIEW_FINDKEY               32813
-#define ID_TREE_EXPANDONELEVEL        32814
 #define ID_TREE_EXPANDBRANCH          32815
-#define ID_TREE_EXPANDALL             32816
 #define ID_TREE_COLLAPSEBRANCH        32817
-#define ID_EDIT_ADDKEY                32818
-#define ID_EDIT_ADDVALUE              32819
-#define ID_EDIT_BINARY                32821
-#define ID_EDIT_STRING                32822
-#define ID_EDIT_DWORD                 32823
-#define ID_EDIT_MULTISTRING           32824
 #define ID_REGISTRY_OPENLOCAL         32825
```

Comment:
Unused: opening the local registry is the default action when regedit is opened, and associated code can be removed too:
https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/regedit/framewnd.c;hb=7dd081e945a1d3756666c2e4c32b32ac74fe7b03#l1372
```c
1372     case ID_REGISTRY_OPENLOCAL:
1373         break;
```

Reply:
I will do that.

Reply:
I acted on that suggestion by https://github.com/reactos/reactos/pull/7194/commits/944fc981be31eab376159c6d137fa92c91ced8fe -> Conversation resolved.